### PR TITLE
Change to fixture

### PR DESCRIPTION
**Pull request by k9ert**
The test-method was quite long and had code-duplication. This is moving it to a fixture style and hopefully speeds it up as well.

### CODE REVIEW

**Comment by relativisticelectron** [comment]
Thanks, I will review it soon.

File: tests/devices_and_wallets.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    return funded_hot_wallet_1
+
+
+@pytest.fixture
+def funded_hot_wallet_2(
+    bitcoin_regtest: NodeController, unfunded_hot_wallet_2: Wallet
+) -> Wallet:
+    funded_hot_wallet_2 = unfunded_hot_wallet_2
+    bitcoin_regtest.testcoin_faucet(funded_hot_wallet_2.getnewaddress())
+    return funded_hot_wallet_2
```
Comment (relativisticelectron) [inline tests/devices_and_wallets.py:99]: This doesn't seem to be used anywhere. Can we remove it?  Or do you intent to use it in the future?

File: tests/test_util_tx.py [hunk 2]
```diff
@@ -2,204 +2,96 @@
...
-            signed = psbtFF
-            for device in devices:
-                signed = device.sign_psbt(signed["psbt"], wallet)
-                if signed["complete"]:
-                    break
-            logging.info(f"signed  {psbtFF}")
-            assert signed["complete"]
-            return signed
+        # fund it
+        bitcoin_regtest.testcoin_faucet(wallet.getnewaddress(), amount=3)
```
Comment (relativisticelectron) [inline tests/test_util_tx.py:79]: Thanks! I didn't know `testcoin_faucet` existed already  :-)

File: tests/devices_and_wallets.py (hunk 1 above)
Comment (k9ert) [reply tests/devices_and_wallets.py:99]: These are fixtures and i think they are usefull for other tests as well. Let's keep them.

**Pull request merged**